### PR TITLE
Allow v5 as peerDependency in ata package

### PR DESCRIPTION
**Pull request by curran**
One small step towards #2807

See comment https://github.com/microsoft/TypeScript-Website/issues/2807

Should in theory solve downstream issue https://github.com/vizhub-core/vzcode/issues/352

### CODE REVIEW

File: packages/ata/package.json [hunk 1]
```diff
@@ -26,6 +26,6 @@
     "jest": "^29.5.0"
   },
   "peerDependencies": {
-    "typescript": "^4.4.4"
+    "typescript": ">=4.4.4 <6.0.0"
```
Comment (jakebailey) [inline packages/ata/package.json]: I wouldn't bother setting an upper bound; TypeScript doesn't generally follow semver in this way so 6 is just going to follow 5.9.

Comment (curran) [reply packages/ata/package.json]: Oh nice! Happy to change it.

Comment (curran) [reply packages/ata/package.json]: Updated

**Review by curran — COMMENTED** [review]
Thanks for reviewing! I removed the upper bound.

**Comment by curran** [comment]
CI failed.
image

It looks like I need to `yarn install` to update the lockfile, as part of the PR. Sorry about that!

**Review by jakebailey — APPROVED** [review]

**Comment by curran** [comment]
@microsoft-github-policy-service agree

**Comment by curran** [comment]
Thanks for taking this over the finish line! Much appreciated.

**Pull request merged**